ludwig/datasets/__init__.py

```diff
@@ -150,12 +236,18 @@ def get_datasets_output_features(
     """
```

Comment:
Let's extend this docstring to include that hugging face datasets are loaded dynamically through a shared connector, so since there is no fixed output feature, HF datasets are excluded.